pootle/apps/pootle_statistics/models.py

```diff
@@ -638,13 +638,17 @@ def get_paid_wordcounts(self):
         """
         ns = self.wordcount
         s = self.get_similarity()
+
+        # translated_words = (ns * (1 - s) * raw_rate + ns * review_rate) / rate
```

Comment:
If the aim is to explain this, you may wanna add a description for humans — FWIW the formula can already be read in the code.